test/src/module/command/controlTest.c

```diff
@@ -51,12 +60,309 @@ testRun(void)
+                ioWriteLine(write, bufNew(0));
+                ioWriteFlush(write);
+            }
+            HARNESS_FORK_PARENT_END();
+        }
+        HARNESS_FORK_END();
+
+        // lock file with another process lock, processId is valid
+        // -------------------------------------------------------------------------------------------------------------------------
+        TEST_RESULT_VOID(storageRemoveNP(storageTest, stanzaStopFile), "remove stop file");
+        HARNESS_FORK_BEGIN()
+        {
+            HARNESS_FORK_CHILD_BEGIN(0, true)
+            {
+                IoRead *read = ioHandleReadNew(strNew("child read"), HARNESS_FORK_CHILD_READ(), 2000);
+                ioReadOpen(read);
+                IoWrite *write = ioHandleWriteNew(strNew("child write"), HARNESS_FORK_CHILD_WRITE());
+                ioWriteOpen(write);
+
+                TEST_RESULT_BOOL(
+                    lockAcquire(lockPath, cfgOptionStr(cfgOptStanza), 0, 30000, true), true,"    child process aquires lock");
+
+                // Let the parent know the lock has been acquired and wait for the parent to allow lock release
+                ioWriteStrLine(write, strNew(""));
+                // All writes are buffered so need to flush because buffer is not full
+                ioWriteFlush(write);
+                // Wait for a linefeed from the parent ioWriteLine below
+                ioReadLine(read);
+
+                lockRelease(true);
+            }
+            HARNESS_FORK_CHILD_END();
+
+            HARNESS_FORK_PARENT_BEGIN()
+            {
+                IoRead *read = ioHandleReadNew(strNew("parent read"), HARNESS_FORK_PARENT_READ_PROCESS(0), 2000);
+                ioReadOpen(read);
+                IoWrite *write = ioHandleWriteNew(strNew("parent write"), HARNESS_FORK_PARENT_WRITE_PROCESS(0));
+                ioWriteOpen(write);
+
+                // Wait for the child to acquire the lock
+                ioReadLine(read);
+
+                TEST_RESULT_VOID(
+                    cmdStop(),
+                    "    stanza, create stop file, force - lock file with another process lock, processId is valid");
+
+                harnessLogResult(strPtr(strNewFmt("P00   INFO: sent term signal to process %d", HARNESS_FORK_PROCESS_ID(0))));
+
+                // Notify the child to release the lock
+                ioWriteLine(write, bufNew(0));
+                ioWriteFlush(write);
+            }
+            HARNESS_FORK_PARENT_END();
+        }
+        HARNESS_FORK_END();
+
+        // lock file with another process lock, processId is invalid
+        // -------------------------------------------------------------------------------------------------------------------------
+        TEST_RESULT_VOID(storageRemoveNP(storageTest, stanzaStopFile), "remove stop file");
+        TEST_RESULT_VOID(
+            storagePutNP(storageNewWriteNP(storageTest, strNewFmt("%s/badpid" LOCK_FILE_EXT, strPtr(lockPath))), BUFSTRDEF("32768")),
```

Comment:
32768 is a valid process id, so this may have unintended consequences, i.e. killing a random other process.  Let's use -32768 instead.